src/stardict_lib.cpp

```diff
@@ -833,21 +833,37 @@ bool SynFile::load(const std::string &url, gulong wc)
 {
     struct stat stat_buf;
     if (!stat(url.c_str(), &stat_buf)) {
-        MapFile syn;
-        if (!syn.open(url.c_str(), stat_buf.st_size))
+
+        FILE *in = fopen(url.c_str(), "rb");
```

Comment:
Why replacing MapFile? Instead of: fopen+fseek,fread and so on,
you can just use `memcpy`

Reply:
Good suggestion.  I dropped the fcalls and switched back to MapFile.